Script/char.dat

```diff
@@ -3343,7 +3929,21 @@ imp
     3,
     "\"And she said: 'Stop pecking at your food!' And I was like: 'But mom! It's raw and I told you I wanted the toddler well done!'\"",
     "\"I was about to kiss her, but then I got summoned.\"",
-    "\"And the summoner tore his heart out and said: 'I present this humble sacrifice to thee, oh daemon of the Nether Realms!' And I was like: 'Gee, that's so sweet and all, but sorry, I already have a valentine.' Plus he was like what, seventy, maybe? I'm ten times older than him! How creepy is that, right?\"";
+    "\"And the summoner tore his heart out and said: 'I present this humble sacrifice to thee, oh daemon of the Nether Realms!' And I was like: 'Gee, that's so sweet and all, but sorry, I already have a date for Valentine.' Plus he was like what, seventy, maybe? I'm ten times older than him! How creepy is that, right?\"";
```

Comment:
*googles* OK, it's a capital V in either case. But the day is `Valentine's` and a person is a `Valentine`.